### DISCUSSION TITLE
Is it possible to favor one radiator over others?

### DISCUSSION THREAD

**Discussion by prigal** [jmcollin78/versatile_thermostat#764] [General]
Hello,

In my living room I have 3 radiators. A main one 1500w, and 2 1000w. The main one is often cold or not really hot while others are very hot. 

They all are attached to the same vtherm.

Problem is the main one is central in the room while the 2 other are closer to the entrance.

Thanks

**Comment by jmcollin78** [comment]
Hello,

no out of the box this is not possible. You have to create more than one VTherm if you need this.

**Reply by prigal** [reply]
Thanks.
